src/main/java/frc/team3388/robot/RobotMap.java

```diff
@@ -14,6 +15,7 @@
     public static final int INTAKE_SYSTEM_MOTOR = 1;
     public static final int PISTON_REVERSE_CHANNEL = 2;
     public static final int PISTON_FORWARD_CHANNEL = 3;
+    public static final int FEEDERSYSTEM =  1;
```

Comment:
Separate the two words with an underscore. Constant names need an underscore:

```java
public static final int FEEDER_SYSTEM = 1;
```

Reply:
סיימתי